src/libraries/System.Runtime/tests/System/DelegateTests.cs

```diff
@@ -102,6 +102,28 @@ public static void DynamicInvoke()
             emptyDelegate.DynamicInvoke(null);
         }
 
+        private class SomeCustomConstantAttribute : CustomConstantAttribute
+        {
+            private const string DEFAULT_VALUE = "SomeValue";
+
+            public static void Do(object o)
+            {
+                //Check the default value is assigned to the parameter as expected
+                Assert.Equal(DEFAULT_VALUE, o);
```

Comment:
Nit: Maybe make this `public static object Do(object o) => o;` and put the assert in the test method itself. I think we tend to put asserts in the `[Fact]` methods themselves.

Reply:
Looks like a good idea to me. I'm going to change it.